### ISSUE TITLE
Managed Kubernetes: CreateCluster: Support for "addons" 

### ISSUE THREAD

**Issue by arafato** [terraform-providers/terraform-provider-alicloud#2103]
Hi--,

can you please add support for the addons attribute documented here:
https://www.alibabacloud.com/help/doc-detail/90776.htm

Thanks, best regards
--Oliver.

**Comment by ringtail** [comment]
@arafato release 1.75.0 support this feature.

**Comment by JanWaller** [comment]
How to I have to configure my terraform script to work with an old cluster (created before this update) to make no changes at all?
* Import does not import the used addons!
* There seem to be differences between the addon documentation for the api and the terraform documentation
* What are my terway pod vswitches? Where they auto created in the old plugin version? Where do I get them from? How to import them?
* Why doesn't it seem to apply the vswitch_ids (worker and pod) when apply? At elast they are not listed under changes?!

**Comment by JanWaller** [comment]
Furthermore, in documentation and examples (and even in code?) you add to the managed cluster lots of stuff only important (and existing) for the normal cluster!

**Comment by ringtail** [comment]
@JanWaller addon is only for creation now.

**Comment by ringtail** [comment]
> Furthermore, in documentation and examples (and even in code?) you add to the managed cluster lots of stuff only important (and existing) for the normal cluster!

I'll check the docs

**Comment by JanWaller** [comment]
So, how do I create a new cluster, that exactly behaves as my old one...?

**Comment by ringtail** [comment]
> So, how do I create a new cluster, that exactly behaves as my old one...?

You mean the addons like before?

**Comment by JanWaller** [comment]
> 
> 
> > So, how do I create a new cluster, that exactly behaves as my old one...?
> 
> You mean the addons like before?

yes exactly. And also for pod_vswitch_ids (that must have been created autoamtically before)

**Comment by ringtail** [comment]
> * What are my terway pod vswitches? Where they auto created in the old plugin version? Where do I get them from? How to import them?

2. I think you can not create a terway network before. So you use pod_cidr instead with flannel network?

**Comment by ringtail** [comment]
If you want to use flannel you can leave pod_vswitch_ids to empty and pass flannel to addons like this. 
```
// Flannel 
    variable "cluster_addons" {
        description = "Addon components in kubernetes cluster"
    
        type = list(object({
            name      = string
            config    = string
        }))
    
        default = [
            {
                "name"     = "flannel",
                "config"   = "",
            },
            {
                "name"     = "flexvolume",
                "config"   = "",
            },
            {
                "name"     = "alicloud-disk-controller",
                "config"   = "",
            },
            {
                "name"     = "logtail-ds",
                "config"   = "{\"IngressDashboardEnabled\":\"true\"}",
            },
            {
                "name"     = "nginx-ingress-controller",
                "config"   = "{\"IngressSlbNetworkType\":\"internet\"}",
            },
        ]
    }
```
